src/main/java/ch/njol/skript/entity/EntityData.java

```diff
@@ -496,6 +497,23 @@ else if(e instanceof PigZombie)
 		return list.toArray((E[]) Array.newInstance(type, list.size()));
 	}
 	
+	@SuppressWarnings({"null", "unchecked"})
```

Comment:
What causes these warnings?

Reply:
I can remove the null one (builds successfully), but the unchecked one was for `unchecked cast Entity to E`

Reply:
Are you sure it builds successfully? CI failed the commit.